.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request, workflow_call, workflow_dispatch]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set CACHE_DATE
+      run: eval echo 'CACHE_DATE="$(date --utc +%Y%m%dT%H%M%SZ)"' >> $GITHUB_ENV
+    - run: ./devel/pull
+    - run: ./devel/build
+    - if: ${{ github.event_name == 'push' }}
```

Comment:
Conditioning this step and the two below on `github.event_name == 'push'` means that images built by `workflow_dispatch` events won't be pushed to Docker Hub.

Reply:
Updated to condition on `github.event_name != 'pull_request'`.